app/labor/markdown_parser.rb

```diff
@@ -6,8 +6,8 @@ def initialize(content)
     @content = content
   end
 
-  def finalize
-    renderer = Redcarpet::Render::HTMLRouge.new(hard_wrap: true, filter_html: false)
+  def finalize(options = { hard_wrap: true, filter_html: false })
```

Comment:
I think this method should only accept `link_attributes` for now, since the rest is the same. Then inside the method you merge `{hard_wrap: true, filter_html: false}` with that.

```suggestion
def finalize(link_attributes: {})
```